packages/prop-house-webapp/src/components/DeleteProposalModal/DeleteProposalModal.module.css

```diff
@@ -0,0 +1,89 @@
+.modal {
```

Comment:
We REALLY REALLY need to refactor all these modals into one. We have this css file repeated prob 5+ times now. 

Let's prioritize after PR merged 🙏

Reply:
will do